Assets/Scripts/Models/Furniture.cs

```diff
@@ -244,28 +291,53 @@ public List<string> ReplaceableFurniture
         }
     }
 
-    // This is a multipler. So a value of "2" here, means you move twice as slowly (i.e. at half speed)
-    // Tile types and other environmental effects may be combined.
-    // For example, a "rough" tile (cost of 2) with a table (cost of 3) that is on fire (cost of 3)
-    // would have a total movement cost of (2+3+3 = 8), so you'd move through this tile at 1/8th normal speed.
-    // SPECIAL: If movementCost = 0, then this tile is impassible. (e.g. a wall).
+    /// <summary>
+    /// Gets the movement cost multiplier that this furniture has. 1 is default, 2 is twice as slow.
+    /// Tile types and environmental effects will be combined with this value (additive).
+    /// If this value is '0' then the furniture is impassable.
```

Comment:
It's a scoped example. It limit the understanding and seems obvious reading the code, but the goal of the documentation is to explain it easily to someone not reading the code.

We should really add that it can be any float value from 0 to any positive number. Negative are not valid, and 0 is a special case